src/sentry/static/sentry/app/views/organizationGroupDetails/eventToolbar.tsx

```diff
@@ -178,13 +182,25 @@ class GroupEventToolbar extends React.Component<Props> {
         </Tooltip>
         {hasQuickTraceView && (
           <LinkContainer>
-            <Link
-              to={generateTraceTarget(evt, organization)}
-              onClick={() => this.handleTraceLink(organization)}
+            <Feature
+              features={['trace-view-summary']}
+              hookName="feature-disabled:trace-view-link"
+              renderDisabled={renderDisabledHoverCard}
             >
-              {' '}
-              View Full Trace
-            </Link>
+              {({hasFeature}) =>
+                hasFeature ? (
+                  <Link
+                    to={generateTraceTarget(evt, organization)}
+                    onClick={() => this.handleTraceLink(organization)}
+                  >
+                    {' '}
```

Comment:
Why is there a leading space?

Reply:
Ah... leftovers from before I realized I needed to use CSS to split it from the event id on the left. Removed 👍